drf_spectacular/views.py

```diff
@@ -1,3 +1,5 @@
+import json
+
```

Comment:
remove the empty line so that isort is happy and the build is green.

https://travis-ci.org/github/tfranzel/drf-spectacular/jobs/734353456#L241

Reply:
Thanks a million!, I'll be up in a minute